src/components/RepaymentScheduleTable.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FunctionComponent } from "react";
+import { TableProps as AntdTableProps } from "antd/lib/table";
+import { ColumnsProps, Table } from "./Table";
+import { Button } from "./Button";
+
+export enum RepaymentScheduleAmountType {
+	INTEREST = "interest",
+	PRINCIPAL = "principal",
+}
+
+export interface RepaymentScheduleDataPoint {
+	month: string;
+	type: RepaymentScheduleAmountType;
+	amount: number;
+}
+
+interface RepaymentScheduleTableProps {
+	dataSource: AntdTableProps<any>["dataSource"];
```

Comment:
Use the props of our own table

Reply:
We can also avoid the any here as we know what kind of data we will be representing here. We can also avoid it in our Table by making the component generic